### PR TITLE
Add support for mountOptions in drive add command

### PR DESCRIPTION
**Pull request by nitisht**
Also update the drive list output table to show
servers in a separate column.

### CODE REVIEW

**Comment by wlan0** [comment]
@nitisht looks good overall. You need to regenerate the CRDs using ./hack/update-codegen.sh

**Review by wlan0 — CHANGES_REQUESTED** [review]
CRD spec needs to be regenerated using ./hack/update-codegen.sh

**Comment by nitisht** [comment]
> CRD spec needs to be regenerated using ./hack/update-codegen.sh

Done

**Comment by wlan0** [comment]
LGTM

File: config/crd/direct.csi.min.io_directcsidrives.yaml [hunk 1]
```diff
@@ -15,85 +15,86 @@ spec:
...
+          path:
+            type: string
+          requestedFormat:
+            properties:
+              filesystem:
+                type: string
+              force:
+                type: boolean
+              mountoptions:
+                type: string
```
Comment (wlan0) [inline config/crd/direct.csi.min.io_directcsidrives.yaml:72]: this should be list

**Review by wlan0 — CHANGES_REQUESTED** [review]

**Review by wlan0 — APPROVED** [review]

**Pull request merged**